### PR TITLE
docs: Banner + readme changes

### PR DESCRIPTION
**Pull request by Penguax**
Added a banner to the top replacing `# Capsule Farmer Evolved` But still contains it as a fallback if image doesn't load. 
Added img.shields icons for repo, linked to leagueofporo already. 
Added readme contents table.
Moved discord and yt links up. Inside of Community Section
Organized Installation and added additional wording for windows installation. 
Corrected `--rm` in docker commands
Added disclaimer at bottom.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,15 +93,15 @@ Pre-built image:
...
+docker run -it --restart unless-stopped --name CapsuleFarmer -d -v /path/to/config.yaml:/config/config.yaml  leagueofporo/capsulefarmer:master
 ```
 
 If you want to build the image locally:
 1. Clone this repo and move to it's direcotry
-2. Build the image: `docker build -t capsulefarmerevolved .`
+2. Build the image: `docker build -t capsulefarmerevolved`.
 3. Edit the `/path/to/config.yaml` to absolute path to your configuration file and run the container in the background:
 ```docker
-docker run -it --rm --restart unless-stopped -d -v /path/to/config.yaml:/config/config.yaml  capsulefarmerevolved
```
Comment (auroraisluna) [inline README.md:105]: Why remove the rm?

**Review by auroraisluna — CHANGES_REQUESTED** [review]

**Comment by auroraisluna** [comment]
Also, forgot to mention, remove the installation guides etc, and link the quickstart to the wiki. As well as the advanced installation/build your self, link to the wiki as well :)

**Comment by Penguax** [comment]
[Advanced Installation For Linux.md](https://github.com/LeagueOfPoro/CapsuleFarmerEvolved/files/10679816/Advanced.Installation.For.Linux.md)
[Advanced Installation for Windows.md](https://github.com/LeagueOfPoro/CapsuleFarmerEvolved/files/10679817/Advanced.Installation.for.Windows.md)
[Advanced Installation for Docker.md](https://github.com/LeagueOfPoro/CapsuleFarmerEvolved/files/10679818/Advanced.Installation.for.Docker.md)

@auroraisluna 
Wiki pages in markdown attached, Readme updated to remove the guides and link to wiki, google tv is included with the correct link if the PR is merged.

**Comment by Penguax** [comment]
[Advanced Installation for Synology Docker.md](https://github.com/LeagueOfPoro/CapsuleFarmerEvolved/files/10679981/Advanced.Installation.for.Synology.Docker.md)
[Advanced Installation for Windows.md](https://github.com/LeagueOfPoro/CapsuleFarmerEvolved/files/10679982/Advanced.Installation.for.Windows.md)
[Advanced Installation for Docker.md](https://github.com/LeagueOfPoro/CapsuleFarmerEvolved/files/10679983/Advanced.Installation.for.Docker.md)
[Advanced Installation for Linux.md](https://github.com/LeagueOfPoro/CapsuleFarmerEvolved/files/10679984/Advanced.Installation.for.Linux.md)


Configuration link was done incorrectly, is now hard linked to wiki link for leagueofporo/capsulefarmerevolved. 

Added Synology Docker readme

**Review by auroraisluna — APPROVED** [review]
LGTM

**Comment by auroraisluna** [comment]
Wiki changes are being pushed as we speak. Wait for @LeagueOfPoro to accept and merge.

**Comment by LeagueOfPoro** [comment]
Looks good, merging

**Pull request merged**